src/index.js

```diff
@@ -14,62 +14,71 @@ var FORMATS = {
 	year: 'yyyy'
 };
 
-function create(time) {
-	return DateTime.fromMillis(time);
-}
-
 _adapters._date.override({
 	_id: 'luxon', // DEBUG
 
+	/**
+	 * @private
+	 */
+	_create: function(time) {
+		return DateTime.fromMillis(time, this.options);
+	},
+
 	formats: function() {
 		return FORMATS;
 	},
 
 	parse: function(value, format) {
+		var me = this;
+
 		if (helpers.isNullOrUndef(value)) {
 			return null;
 		}
 
 		var type = typeof value;
 		if (type === 'number') {
-			value = create(value);
+			value = me._create(value);
 		} else if (type === 'string') {
 			if (typeof format === 'string') {
-				value = DateTime.fromFormat(value, format);
+				value = DateTime.fromFormat(value, format, me.options);
 			} else {
-				value = DateTime.fromISO(value);
+				value = DateTime.fromISO(value, me.options);
 			}
 		} else if (type === 'object') {
 			value = DateTime.fromObject(value);
 		} else if (value instanceof Date) {
-			value = DateTime.fromJSDate(value);
+			value = DateTime.fromJSDate(value, me.options);
 		}
 
 		return value.isValid ? value.valueOf() : null;
 	},
 
 	format: function(time, format) {
-		return create(time).toFormat(format);
+		var me = this;
+		return me._create(time).toFormat(format, me.options);
```

Comment:
Caching `this` as `me` is interesting starting 4 uses of `this` (or if we actually need to capture `this` in a callback). In this case, I would use `this` directly.

Reply:
Actually, Terser will not use `me` until 5 replacements, though here, I think it's nicer to use `this` directly.